### ISSUE TITLE
icon request: ruler

### ISSUE THREAD

**Issue by justinlogiudice** [ionic-team/ionicons#1171]
### Describe the Icon

As an example, a ruler icon could be used when users are setting distance unit preferences between miles and kilometers

**Comment by Symo254-archi** [comment]
Yes, a ruler icon is often used to represent measurements and could be used to indicate distance unit preferences. This allows users to easily understand the context and purpose of the icon, making it a clear and intuitive choice.

**Comment by christian-bromann** [comment]
Consolidating into #1449
